runtime/balrt.h

```diff
@@ -67,6 +67,11 @@ typedef int64_t CompareResult;
 #define IMMEDIATE_INT_MAX  (((int64_t)1 << (TAG_SHIFT - 1)) - 1)
 #define IMMEDIATE_INT_TRUNCATE(n) (n & (((int64_t)1 << TAG_SHIFT) - 1))
 
+#define TAGGED_FALSE (((uint64_t)TAG_BOOLEAN) << TAG_SHIFT)
+#define TAGGED_TRUE ((((uint64_t)TAG_BOOLEAN) << TAG_SHIFT) | 1)
+#define TAGGED_INT_ZERO ((((uint64_t)TAG_INT) << TAG_SHIFT) | IMMEDIATE_FLAG)
```

Comment:
IMMEDIATE_FLAG at the beginning for consistency

Reply:
fixed